test/org/sagebionetworks/bridge/services/AuthenticationServiceMockTest.java

```diff
@@ -735,8 +738,107 @@ public void creatingExternalIdOnlyAccountSucceedsIfIdsManaged() {
         
         StudyParticipant participant = new StudyParticipant.Builder().copyOf(PARTICIPANT)
                 .withEmail(null).withPhone(null).withExternalId("id").build();
-        service.signUp(study, participant, false);
+        service.signUp(study, participant);
         
         verify(participantService).createParticipant(study, NO_CALLER_ROLES, participant, true);
     }
-}
+    
+    @Test
+    public void signInWithIntentToParticipate() throws Exception {
+        account.setId(USER_ID);
+        Account consentedAccount = new GenericAccount();
```

Comment:
We should create a Account.create() so we don't have references to GenericAccount everywhere. (Here and several other places in this PR.)

Reply:
Done.